### ISSUE TITLE
search for default browser instead of google chrome.

### ISSUE THREAD

**Issue by ekinertac** [OthersideAI/self-operating-computer#6]

**Comment by ElCuboNegro** [comment]
do you have any idea on how to describe the issue?

**Comment by ekinertac** [comment]
I thought the title was quite self-explanatory, but let me simplify it further.

The 'operate' command should look for the operating system's default browser, rather than trying (and failing) to find Google Chrome, particularly for those who haven't installed Google Chrome on their computers.

Do you get it now?

**Comment by Daisuke134** [comment]
@joshbickett I agree. I am not sure how much using different browsers would affect the SOC performance, but I think that this is definitely necessary for those who have not installed Chrome on their device.

**Comment by joshbickett** [comment]
@Daisuke134 yeah, performance is my main concern. I wouldn't want to sacrifice the experience for the many (since I think chrome is over 60% of market if I remember correctly). Long-term to build in local settings memory to the project. On the first search for Chrome doesn't yield anything then the agent could save that to local memory and look for another browser to use in the future.

**Comment by VishnuPonnusamy** [comment]
i tried to run and quits after saying downloading chrome . 
how can i resolve this issue
